controllers/api/userRoutes.js

```diff
@@ -61,16 +61,15 @@ router.post('/', (req, res) => {
     User.create({
       email: req.body.email,
       password: req.body.password,
-      name: req.body.name,
-      last_name: req.body.last_name,
-      role: req.body.role
+      username: req.body.username,
```

Comment:
there is no username field, aren't we just using email to log on?

Reply:
commented out